libs/domain/search/src/services/search.providers.ts

```diff
@@ -44,3 +46,10 @@ export const searchProviders: Provider[] = [
     useClass: SearchPageTitleMetaResolver,
   },
 ];
+
+export const searchPreviewProviders: Provider[] = [
+  {
+    provide: ExperienceDataRevealer,
```

Comment:
I prefer to not hang the revealer up on Experience Builder, as other integrations can benefit from this as well.

Reply:
I do like this revealer pattern though!